package.json

```diff
@@ -10,8 +10,10 @@
     "index.c"
   ],
   "scripts": {
-    "install": "node-gyp-build",
-    "test": "node test.js"
+    "install": "prebuild-install || node-gyp-build",
+    "test": "node test.js",
+    "prebuild": "prebuild -t 9.0.0 --strip -u $githubtoken",
+    "prepublishOnly": "npm run prebuild"
```

Comment:
This should be `npm test && npm run prebuild` in order to prevent publishing a broken version.

Reply:
thanks